### PR TITLE
Add transformers dependency and update notebooks

### PR DESCRIPTION
**Pull request by bsuryadevara**
Added missing dependencies and updated broken notebooks.

### CODE REVIEW

**Comment by bsuryadevara** [comment]
@gbatmaz can we upload `./labelled_nv_smi.json` dataset to S3?
reference: https://github.com/bsuryadevara/clx/blob/update-notebooks/notebooks/anomalous_behavior_profiling/anomalous_behavior_profiling_supervised.ipynb

**Review by tzemicheal — APPROVED** [review]

**Review by efajardo-nv — APPROVED** [review]

**Comment by gbatmaz** [comment]
> @gbatmaz can we upload `./labelled_nv_smi.json` dataset to S3?
> reference: https://github.com/bsuryadevara/clx/blob/update-notebooks/notebooks/anomalous_behavior_profiling/anomalous_behavior_profiling_supervised.ipynb

Hi Bhargav. Can you please not add any data as discussed. Many thanks.

**Review by gbatmaz — APPROVED** [review]

**Comment by bsuryadevara** [comment]
@ajschmidt8 @efajardo-nv when you have time, could you kindly verify these changes and merge?

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -39,11 +39,15 @@ gpuci_logger "Activate conda env"
...
-gpuci_logger "Install tests dependencies"
+gpuci_logger "Install conda dependencies"
 gpuci_mamba_retry install -y \
     "cuxfilter=${MINOR_VERSION}" \
     "faker" \
     "python-whois" \
+    "requests" \
+    "matplotlib" \
+    "faker" \
+    "transformers=4.*" \
```
Comment (efajardo-nv) [inline ci/gpu/build.sh:50]: don't think we need to add these. `requests`,`matplotlib`, and `transformers` already on the images used for ci builds. `faker` already added.

Comment (bsuryadevara) [reply ci/gpu/build.sh:50]: ok, I will remove them. thanks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -146,8 +146,7 @@ Create the conda development environment:
 ```bash
 # create the conda environment (assuming in base `clx` directory)
 
-conda env create --name clx_dev --file conda/environments/clx_dev_cuda11.0.yml
-
+mamba env update --name clx_dev --file conda/environments/clx_dev_cuda11.5.yml
```
Comment (efajardo-nv) [inline CONTRIBUTING.md:149]: there are other spots in `CONTRIBUTING` and also `README` where we need update from `11.0` to `11.5`.

Comment (efajardo-nv) [reply CONTRIBUTING.md:149]: also should be `mamba env create`

Comment (bsuryadevara) [reply CONTRIBUTING.md]: Sorry, my bad. I missed them. I will update all the spots to cuda11.5. thank you

Comment (bsuryadevara) [reply CONTRIBUTING.md]: other spots were updated. thank you

**Comment by efajardo-nv** [comment]
@gpucibot merge

**Pull request merged**